src/vector_topic_modeling/distributed.py

```
@@ -0,0 +1,187 @@
+    client.delete(clusters_key)
+    client.delete(tasks_key)
+    client.delete(results_key)
+
+    if not results:
+        # Fallback if workers failed
+        return base_metrics
+
+    # Aggregate results into overall silhouette score
+    s_scores = []
+    for s_val in results.values():
+        s_scores.append(float(s_val))
```

Ruff (0.15.7)

[warning] Use a list comprehension to create a transformed list

(PERF401)